<launch>

    <node name="VtkToCloud" pkg="ethzasl_point_cloud_vtk_tools" type="vtkToPointCloud" output="screen">
        <param name="singleFile" type="string" value="$(find tensorvoting)/data/Curveplane.vtk"/>
        <!-- <param name="singleFile" type="string" value="$(find tensorvoting)/data/planes4.vtk"/> -->

    </node>
    <node name="test_plan" pkg="tensorvoting" type="test_plan">
        <param name="sigma" value="1.5"/>
        <param name="cloudTopicIn" value="/point_cloud"/>
        <param name="cloudTopicOut" value="/point_cloud_sparsevoting"/>
        <param name="mapFrameId" value="map"/>
        <param name="savevtk" value="false"/>
    </node>
    <!-- <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find tensorvoting)/launch/rviz_tensorvoting.vcg" /> -->
    <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find tensorvoting)/launch/rviz_tensorvoting.rviz" />
</launch>